UnityProject/Assets/Scripts/Alien/AlienEggCycle.cs

```diff
@@ -24,6 +24,7 @@ public class AlienEggCycle : MonoBehaviour, IInteractable<HandApply>, IServerSpa
 		[SerializeField]
 		private EggState initialState = EggState.Growing;
 
+		// TODO This is entirely unused.
```

Comment:
Should be used on caverna away site unless i missed doing that.

Reply:
@ThatDan123 It's definitely set inside the editor. But the compiler can't find any references to it being *used* anywhere.

We might want to leave the variable for future use, but are you certain it's implemented?